recipes/monotonic/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "monotonic" %}
+{% set version = "1.3" %}
```

Comment:
that was silly of me. I will do that right away.
I was essentially in auto-mode of condo-build -> fail -> add recipe -> repeat.

Thanks for catching it.